flatnav/Index.h

```diff
@@ -518,18 +519,6 @@ template <typename dist_t, typename label_t> class Index {
     float min_dist = std::numeric_limits<float>::max();
     node_id_t entry_node = 0;
 
-    /**
-     * @brief If we use a product quantizer, this is what will happen:
-     * 1. When comparing the query to a database vector (during insertion or
-     * search in HNSW), you don't directly compute a distance using their
-     * original vectors or even use the PQ code of the query. Instead, for each
-     * segment of the database vector, look up the distance between the query's
-     * segment and the database vector's corresponding centroid using the
-     * pre-computed distance table for that segment. Sum these distances across
-     * all segments to get the total asymmetric distance between the query and
-     * the database vector.
-     *
-     */
```

Comment:
Not sure why I thought this comment belonged here in the first place. It was probably when I still had product quantization as a member variable in the index class instead of having it as an implementer of the distance interface.